datapirates/challenge.md

```diff
@@ -0,0 +1,26 @@
+# Data Pirates challenge
+
+Welcome to Data Pirates challenge.
+
+
+## Scenario:
+
+The Internet Movie Database (IMDb) is an online database of information about music, movies, movies, TV shows and games and television games computer. [Wikipedia]. This challenge consist to collect information from this site and deliver it other format.
```

Comment:
movies twice here :)

Reply:
...from this site and deliver it **in** other format.

Reply:
thanks :)